apps/communications/contacts/js/views/form.js

```diff
@@ -835,7 +835,7 @@ contacts.Form = (function() {
   var cookMatchingCallbacks = function cookMatchingCallbacks(contact) {
     return {
       onmatch: function(results) {
-        ExtServices.showDuplicateContacts();
+        MatchService.showDuplicateContacts();
 
         mergeHandler = function mergeHandler(e) {
           if (e.origin !== fb.CONTACTS_APP_ORIGIN) {
```

Comment:
I would remove all code as https://github.com/mozilla-b2g/gaia/pull/30622/files#diff-7d9bc4ef89ed32b6f748a27867f44c2aR863 because not it's not part of FB (It's part of `Matcher`)